src/schema/nmdc.yaml

```diff
@@ -2247,6 +2247,44 @@ enums:
       V-bottom conical tube:
       falcon tube:
 
+  FailureWhatEnum:
```

Comment:
I think we should make a note somewhere about these permissible values that share names with classes. It's especially interesting in the OWL serialization, becasue the bases of the URIs are quite different